specification/containerservice/resource-manager/Microsoft.ContainerService/aks/readme.md

```diff
@@ -45,6 +45,7 @@ These settings apply only when `--tag=package-preview-2023-11` is specified on t
 ```yaml $(tag) == 'package-preview-2023-11'
 input-file:
   - preview/2023-11-02-preview/managedClusters.json
+  - preview/2023-11-02-preview/scheduledEvents.json
```

Comment:
Yes, if we don't want to include this API in the SDK, we should remove the settings here.

Reply:
I do want to publish the data model part of this API: to be more specific, the "ScheduledEvent" object in the PR. I though we need to include the above scheduledEvents.json file so it can be published to SDK.

@FumingZhang Please correct me if I am wrong.

Reply:
But like matt said, will users (even azure notification) use it through the SDK?

Reply:
Will not expose it to SDK.